src/app/home-page/widget/organization-box/organization-box.component.html

```diff
@@ -0,0 +1,139 @@
+          <div [ngPlural]="pendingRequests.length">
+            <ng-template ngPluralCase="=0">0 Pending Requests</ng-template>
+            <ng-template ngPluralCase="=1">1 Pending Request</ng-template>
+            <ng-template ngPluralCase="few">{{ pendingRequests.length }} Pending Requests</ng-template>
+          </div>
+        </a>
+      </div>
+      <div id="pending-invites" fxLayout="row" fxLayoutAlign="center center" class="mb-3">
+        <mat-icon class="pending-invites mr-1">error</mat-icon>
+        <a routerLink="/accounts" class="size-small pending-links">
+          <div [ngPlural]="pendingInvites.length">
+            <ng-template ngPluralCase="=0">0 Pending Invitations</ng-template>
+            <ng-template ngPluralCase="=1">1 Pending Invitation</ng-template>
+            <ng-template ngPluralCase="few">{{ pendingInvites.length }} Pending Invitations</ng-template>
+          </div>
+        </a>
+      </div>
+      <button
+        mat-button
+        class="private-btn small-btn-structure mb-3"
+        id="requestOrganizationButton"
+        matTooltip="Request Organization"
+        matTooltipPosition="after"
+        (click)="requireAccounts()"
+      >
+        <img src="../../../assets/svg/icons-plus.svg" alt="Register an Organization" class="pr-2 pb-1" />
+        Request an Organization
+      </button>
+    </div>
+  </div>
+  <mat-divider></mat-divider>
+  <mat-card-content class="p-3">
+    <app-loading [loading]="isLoading | async">
+      <div *ngIf="events.length !== 0 || isLoading; else noEvents">
+        <h5 class="weight-bold">Recent Activity</h5>
+        <div class="size-small list-events" *ngFor="let event of events" fxLayout="row" fxLayoutAlign="space-between center">
+          <div fxLayout="row" fxLayoutAlign="start center">
+            <img src="{{ event?.initiatorUser?.avatarUrl }}" alt="User avatar" />
+            <div [ngSwitch]="event.type">
+              <div *ngSwitchCase="EventType.CREATECOLLECTION" class="my-3">
+                <strong>{{ event.initiatorUser?.username }}</strong> created the collection
+                <a [routerLink]="'/organizations/' + event.organization?.name + '/collections/' + event.collection?.name"
+                  >{{ event.collection?.name }}
+                </a>
+                in organization
+                <a [routerLink]="'/organizations/' + event.organization?.name">{{ event.organization?.displayName }}</a>
+                <div class="subtitle">{{ event.dbCreateDate | date: 'medium' }}</div>
+              </div>
+              <div *ngSwitchCase="EventType.ADDTOCOLLECTION" class="my-3">
+                <strong>{{ event.initiatorUser.username }} </strong> added the {{ event?.tool || event?.apptool ? 'tool' : 'workflow' }}
+                <a
+                  [routerLink]="
+                    event?.apptool
+                      ? '/tools/' + event?.apptool.full_workflow_path
```

Comment:
Not sure if this is right. See https://github.com/dockstore/dockstore-ui2/blob/ff13411f0e34898a183263f8ba6a81a4e002d3d5/src/app/entry/router-link.pipe.ts#L19

Ideally you would use that pipe, but it only handles workflows, app tools, and services, and I think you need to handle tools as well.